dataprofiler/profilers/profiler_options.py

```diff
@@ -1133,6 +1133,7 @@ def __init__(
         self,
         null_values: dict[str, re.RegexFlag | int] = None,
         column_null_values: dict[int, dict[str, re.RegexFlag | int]] = None,
+        sampling_ratio: float = None,
```

Comment:
let's set the default to `0.2` actually. To match the current default setting in the core library [here](https://github.com/capitalone/DataProfiler/blob/df86943d2f06610f51c07e33e8e990ed2ff54e1b/dataprofiler/profilers/base_column_profilers.py#L28)